crates/ruff_linter/src/rules/airflow/rules/removal_in_3.rs

```diff
@@ -203,6 +204,54 @@ fn check_call_arguments(checker: &mut Checker, qualname: &QualifiedName, argumen
     };
 }
 
+/// Check whether a removed context key is access through context.get("key").
+///
+/// ```python
+/// from airflow.decorators import task
+///
+///
+/// @task
+/// def access_invalid_key_task_out_of_dag(**context):
+///     print("access invalid key", context.get("conf"))
+/// ```
+fn check_context_get(checker: &mut Checker, call_expr: &ExprCall) {
+    const REMOVED_CONTEXT_KEYS: [&str; 1] = ["conf"];
```

Comment:
How many arguments are you planning to deprecate for this to be a list?

Reply:
I think there are multiple as stated in this PR description: https://github.com/astral-sh/ruff/pull/15144#issue-2759448217